app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/SearchResultsAdapter.java

```diff
@@ -53,7 +53,7 @@ public RssViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     @Override
     public void onBindViewHolder(final RssViewHolder holder, int position) {
         Datum datum = datumList.get(position);
-        if (datum != null) {
```

Comment:
Yes, i think so.
 I am not that familiar with the codebase but this solved the crash that was occurring before.